sources/rpmbootstrap.go

```diff
@@ -28,7 +34,7 @@ func (s *rpmbootstrap) repodirs() (dir string, err error) {
 	distribution := s.definition.Image.Distribution
 	content := s.definition.Source.URL
 	if distribution == "" || content == "" {
-		err = fmt.Errorf("No valid distribution and source url specified")
+		err = fmt.Errorf("no valid distribution and source url specified")
```

Comment:
We like our errors to start with a capital letter (even if Go upstream doesn't like it).